src/main/kotlin/turfkt/CenterOfMass.kt

```diff
@@ -0,0 +1,101 @@
+import geojsonkt.*
```

Comment:
This file does not compile. You'll have to get it at least compiling before this PR can be merged.